specification/columns/chargecategory.md

```diff
@@ -33,10 +30,11 @@ Allowed values:
 
 | Value      | Description                          |
 | :--------- | :------------------------------------|
-| Adjustment | Any adjustments that are applied after the original usage or purchase row. Adjustments may be related to multiple charges.   |
+| Usage      | Charges based on the quantity of a service or resource that was consumed over a given period of time.     |
 | Purchase   | Charges for the acquisition of a service or resource bought upfront or on a recurring basis.              |
 | Tax        | Applicable taxes that are levied by the relevant authorities. Tax charges may vary depending on factors such as the location, jurisdiction, and local or federal regulations. |
-| Usage      | Charges based on the quantity of a service or resource that was consumed over a given period of time.     |
+| Adjustment      | Positive or negative charges the provider applies that do not fall into other category values.    |
+| Credit      | Negative charges granted by the provider for various scenarios, like negotiated benefits, usage discounts, or promotional credits.     |
```

Comment:
Hopefully we can get crisp with our definition here. We need to be clear on when something should come as a 'Credit' vs coming in line with usage/purchase. It would be nice if we could make a statement about SKU level usage and purchase discounts needing to apply inline within those two - so commitment discounts, negotiated discounts, etc. aren't accounted for as credits.

Reply:
updated in call on 29.3